docs/conf/portland/2025/cfp.rst

```diff
@@ -166,6 +175,24 @@ If you already have a ticket, we will of course refund it - just drop us an emai
 
 Note that all Speakers must read, understand, and agree to our :doc:`/code-of-conduct`. All talks and slides will need to follow our Code of Conduct. If you are unsure about any aspect of this, please ask us for clarification.
 
+Timeline overview
+-----------------
+
+**{{cfp.ends}}**
+    Call for Proposal ends
+
+**{{cfp.notification}}**
+    We'll let you know whether your proposal was accepted
+
+**{{cfp.slides_by}}**
+    We'll ask you for a copy of your slides so that our human captioners can prepare for your talk
+
+**{{cfp.info_by}}**
+    We'll ask you for your pronouns, how your pronounce your name and some interesting facts about you so our emcee can prepare to introduce you.
```

Comment:
We'll ask you for your pronouns, how you pronounce your name, and some interesting facts about you so our emcee can prepare to introduce you.